### PR TITLE
Increasing coverage for flask_util

### PR DESCRIPTION
**Pull request by theacodes**
Coverage is at 97%. Apparently I'm missing a few branches somehow, but I'm unsure how to find them.

### CODE REVIEW

**Comment by dhermes** [comment]
To find the branches run:

```bash
$ tox -e cover
$ coverage html
$ open htmlcov/oauth2client_flask_util.html
```

and then view the branch misses as HTML in your browser.

Ping me after they are addressed?

**Comment by jonparrott** [comment]
Thanks for the tip. 100% coverage.

File: oauth2client/flask_util.py [hunk 1]
```diff
@@ -536,9 +536,7 @@ def locked_get(self):
             return None
 
         credentials = OAuth2Credentials.from_json(serialized)
-
-        if credentials:
-            credentials.set_store(self)
+        credentials.set_store(self)
```
Comment (dhermes) [inline oauth2client/flask_util.py:539]: Oof. How did we miss that.

**Comment by dhermes** [comment]
LGTM

@nathanielmanistaatgoogle I'll let you weigh in

**Pull request merged**